integration_tests/CMakeLists.txt

```diff
@@ -392,8 +392,8 @@ RUN(NAME str_to_list_cast    LABELS cpython llvm c)
 RUN(NAME generics_01         LABELS cpython llvm)
 RUN(NAME generics_02         LABELS cpython llvm)
 RUN(NAME generics_array_01   LABELS cpython llvm)
-RUN(NAME generics_array_02   LABELS cpython llvm)
-RUN(NAME generics_array_03   LABELS cpython llvm)
+RUN(NAME generics_array_02   LABELS cpython)
+RUN(NAME generics_array_03   LABELS cpython)
```

Comment:
@ansharlubis Would request you to re-enable ``llvm`` for these tests in this PR.

Reply:
Done. cc: @ansharlubis